src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -95,6 +106,20 @@ function MetadataSidebarRedesign({
         setEditingTemplate(null);
     };
 
+    const isExistingMetadataInstance = (): boolean => {
+        return (
+            editingTemplate && !!templateInstances.find(templateInstance => templateInstance.id === editingTemplate.id)
+        );
+    };
+
+    const handleSubmit = async (values: FormValues, operations: JSONPatchOperations) => {
```

Comment:
Do we need async here? We don't return a value and theres no use of await

Reply:
Yes, `MetadataInstanceForm` component from metadata-editor expects async function